CHANGELOG.md

```diff
@@ -4,19 +4,102 @@
 
 ### Added
 - Merge ([ods-mro-shared-library](https://github.com/opendevstack/ods-mro-jenkins-shared-library)) into this library, and consolidate services ([#271](https://github.com/opendevstack/ods-jenkins-shared-library/issues/271))
-- Introduce CodenArc for scanning of the shared lib ([#290](https://github.com/opendevstack/ods-jenkins-shared-library/issues/290))
-- Central openshift namespace is NOT fixed to 'CD' anymore, it's now configurable ([#311](https://github.com/opendevstack/ods-jenkins-shared-library/issues/311))
-- Support for mono-repos with multiple containers ([#229](https://github.com/opendevstack/ods-jenkins-shared-library/issues/229))
-- Add harmonized stage names based on the new library layout ([#215](https://github.com/opendevstack/ods-jenkins-shared-library/issues/215))
 - Allow configuration of docker context directory ([#181](https://github.com/opendevstack/ods-jenkins-shared-library/issues/181))
+- Add harmonized stage names based on the new library layout ([#215](https://github.com/opendevstack/ods-jenkins-shared-library/issues/215))
+- Introduce CodenArc for scanning of the shared lib ([#290](https://github.com/opendevstack/ods-jenkins-shared-library/issues/290))
+- Add global var withOpenShiftCluster ([#412](https://github.com/opendevstack/ods-jenkins-shared-library/pull/412))
+- Add stage to import image and automatically import if possible ([#400](https://github.com/opendevstack/ods-jenkins-shared-library/pull/400))
+- For agents image configuration - check availability of image & set timeout to connect ([#391](https://github.com/opendevstack/ods-jenkins-shared-library/issues/391))
+- add withStage for quickstarters ([#378](https://github.com/opendevstack/ods-jenkins-shared-library/pull/378))
+- (Optionally) Deploy via Tailor ([#96](https://github.com/opendevstack/ods-jenkins-shared-library/issues/96))
+- Allow configuration of BB project ([#317](https://github.com/opendevstack/ods-jenkins-shared-library/pull/317))
+- Make central namespace configurable ([#311](https://github.com/opendevstack/ods-jenkins-shared-library/issues/311))
+- Make ODS namespace configurable ([#312](https://github.com/opendevstack/ods-jenkins-shared-library/pull/312))
```

Comment:
I think here we have the pull request and the correlating issue (311 and 312). I would list only one of them. I'd probably go for the pull request.